moped-editor/src/components/GridTable/helpers.js

```diff
@@ -155,11 +155,13 @@ export const useMakeFilterState = ({
       const filterSearchParams = searchParams.get(
         advancedSearchFilterParamName
       );
-      if (filterSearchParams === null) return [];
+      if (filterSearchParams === null) {
+        return isEmptyFilterNeeded ? [generateEmptyFilter()] : [];
```

Comment:
awesome! so nice to see this snap into place. could you add `isEmptyFilterNeeded` to the docstring above so we can sneak it in this PR too? maybe something like:

`* @param {Boolean} isEmptyFilterNeeded - toggle adding an empty filter to add initial row in the Filters component`

idk - does that sounds okay to y'all? cc @roseeichelmann